File: A2/A2-026/A2-026.cpp
#include<iostream>
#include<vector>
#include<algorithm>
using namespace std;

int N, fatCount = 0;
vector<pair<int, string>> rabbits;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input number of rabbits
    cin >> N;
    // Input rabbit information
    for(int i = 0; i < N; i++)
    {
        string name;
        int weight;
        cin >> name >> weight;
        rabbits.push_back(make_pair(weight, name));
        // Count fat rabbits
        if(weight > 15) { fatCount++; }
    }
    // Sort rabbit by weights
    sort(rabbits.begin(), rabbits.end());
    // Output
    cout << fatCount << '\n';
    cout << rabbits[N - 1].second << ' ' << rabbits[N - 1].first << '\n';
    return 0;
}

File: A2/A2-026/A2-026.py
# Input rabbit numbers
N = int(input())

# Input rabbit name and weight
rabbits = []
fat_count = 0
for _ in range(N):
    name, weight = input().strip().split()
    rabbit = [int(weight), name]
    rabbits.append(rabbit)
    # Count fat rabbits
    if(int(weight) > 15):
        fat_count += 1

# Sort rabbit by weights in descending order
rabbits.sort(reverse = True)

# Output
print(fat_count)
print(rabbits[0][1], rabbits[0][0])